scripts/save_kube_credentials.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+# The string to match with context.user
+USERNAME="$1"
```

Comment:
MAYBE: we could request just the context name from the user, from which we extract the username and cluster name, to minimize the number of arguments the user has to input.

Reply:
As the context names uniquely identify the cluster and the username

Reply:
Solved by: [c7d25f1](https://github.com/OpenMined/PySyft/pull/8199/commits/c7d25f1d5cffd4c76f7666a27f1d27ada504d56f)